src/Symfony/Component/Message/MessageBusInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface MessageBusInterface
+{
+    /**
+     * Dispatch the given message.
+     *
+     * The bus can return a value coming from handlers, but is not required to do so.
+     *
+     * @param object $message
+     *
+     * @return mixed
+     */
+    public function dispatch($message);
```

Comment:
For the record, this [has been discussed briefly here](https://github.com/symfony/symfony/pull/24411#discussion_r143510824). I'd keep `dispatch` for the simple reason that the bus itself do not "handles" the message, it dispatches it to some handler(s). It allows us (and the developers) to clearly identify that between the moment the message is *dispatched* and the moment the message is *handled* it can go through the wire (i.e. brokers, etc...).